app/models/user.rb

```diff
@@ -92,7 +92,7 @@ def volunteers_serving_transition_aged_youth
     volunteers.includes(
       case_assignments: :casa_case
     ).where(
-      case_assignments: {active: true}, casa_cases: {active: true, transition_aged_youth: true}
+      case_assignments: {active: true}, casa_cases: {active: true, birth_month_year_youth: ..14.years.ago}
```

Comment:
@librod89 could we use a constant for 14, maybe put it in the CasaCase model?

Reply:
Yes, good idea